Lib/test/test_pow.py

```diff
@@ -22,9 +22,10 @@ def powtest(self, type):
             for othertype in (int,):
```

Comment:
Mark is correct: (int,) used to be (int, float). If this line is deleted, other changes below must be applied.  For some reason, I could not make them all at once.
```suggestion
```